### DISCUSSION TITLE
Using Obsidian as my main client to communicate with Claude, without using existing notes

### DISCUSSION THREAD

**Discussion by AndreyLukin-SevenAI** [logancyang/obsidian-copilot#889] [Q&A]
Hi team,

I am looking to use Obsidian Copilot as my go to Q/A platform for all LLM related things. I want to be able to ask pure Claude questions, but I am getting responses that tell me that I dont have the necessary context to answer these questions. Is there a way to directly hit Claude in the copilot?

TIA!
<img width="942" alt="image" src="https://github.com/user-attachments/assets/080f0ccb-d3da-4509-b37b-ee9981b2c61f">

**Comment by logancyang** [comment]
What mode are you in? Vault QA? I can't reproduce your result.
<img width="946" alt="SCR-20241202-lbtg" src="https://github.com/user-attachments/assets/b6b1fd45-87e5-4db6-992b-c56e39da44c7">

**Reply by AndreyLukin-SevenAI** [reply]
Hi @logancyang I have it set just to chat 
<img width="979" alt="image" src="https://github.com/user-attachments/assets/c9a1d34d-3a9b-48ab-8024-9fb623932f0d">

I just set up obsidian for the first time, could I be missing something? I am using claude

**Reply by logancyang** [reply]
@AndreyLukin-SevenAI OK I think this is caused by the default system prompt. Try putting something like "answer to the best of your knowledge" in the system prompt in Copilot settings, click save, and try again. In the next releases I'm going to make it easier out-of-the-box.

**Reply by AndreyLukin-SevenAI** [reply]
Yes, that fixed it! TY. Looking forward for the update
